### ISSUE TITLE
graph flickers in urxvt

### ISSUE THREAD

**Issue by rcoh** [SumoLogic/sumoshell#6]
Not totally sure why, but it doesn't happen in gnome-terminal

**Comment by joeblew99** [comment]
this is now moot, because you use github.com/jroimartin/gocui?

**Comment by rcoh** [comment]
No, the graph library is still used to support the graph binary. However, we don't use curses for `render` anymore so that is no longer an issue
